src/layouts/flex/lv_flex.c

```diff
@@ -395,6 +395,10 @@ static void children_repos(lv_obj_t * cont, flex_t * f, int32_t item_first_id, i
             item = get_next_item(cont, f->rev, &item_first_id);
             continue;
         }
+
+        int16_t item_w_layout = item->w_layout;
+        int16_t item_h_layout = item->h_layout;
```

Comment:
Oh, also, to match the struct member's type

```suggestion
        uint16_t item_w_layout = item->w_layout;
        uint16_t item_h_layout = item->h_layout;
```